nanochat/worker.py

```
@@ -0,0 +1,886 @@
+    parent = str(Path(NANOCHAT_DIR).parent)
+    if parent not in sys.path:
+        sys.path.insert(0, parent)
+    import torch  # noqa: F401
+    _nanochat_imported = True
+
+
+def safe(fn):
+    async def wrapper(data):
+        try:
+            return await fn(data)
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)